.github/workflows/cpu-tests.yml

```diff
@@ -24,8 +24,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - {os: "macOS-11", python-version: "3.10"}
-          - {os: "ubuntu-20.04", python-version: "3.10"}
+          - {os: "macOS-12", python-version: "3.10"}
+          - {os: "ubuntu-22.04", python-version: "3.11"}
+          - {os: "ubuntu-22.04", python-version: "3.10"}
+          - {os: "ubuntu-22.04", python-version: "3.9"}
+          - {os: "ubuntu-20.04", python-version: "3.8"}
```

Comment:
I've checked. 3.8 can be installed in 22.04.
So maybe for uniformity, use 22.04 everywhere?

Reply:
Lets keep granny around until she dies

Reply:
I was talking about Ubuntu.
But I guess it might be a good idea to have old Ubuntu-Python combo as a test runner.